### ISSUE TITLE
feature request. Change the number of frames in the Onion skin function

### ISSUE THREAD

**Issue by AlexKiryanov** [MaurycyLiebner/enve#180]
Good evening!
When working with frame-by-frame animation, I used Onion skin. It’s great that there is such a function, but when you work with line frames that are close to each other, it’s hard to understand where the previous frame is.
It would be great if I could adjust how much before and how many frames to show this option after frames.
Is it possible to do this?

**Comment by calogerogigante** [comment]
Yes, me too, I have the same problem. And I can ask one thing more : can we have the choice of the opacity of the onion skin ? Because, sometimes, you need to see clearly the previous drawing (great opacity), and sometimes, just a light seeing (light opacity)... Thanks in advance....
